src/main/java/org/gridsuite/directory/server/DirectoryService.java

```diff
@@ -244,16 +244,24 @@ public List<ElementAttributes> getDirectoryElements(UUID directoryUuid, String u
             return List.of();
         }
 
-        return getDirectoryElementsStream(directoryUuid, userId, types).collect(Collectors.toList());
+        return getDirectoryElementsStream(directoryUuid, userId, types, stashed).collect(Collectors.toList());
     }
 
     private Stream<ElementAttributes> getDirectoryElementsStream(UUID directoryUuid, String userId, List<String> types) {
-        return getAllDirectoryElementsStream(directoryUuid, types, userId)
+        return getDirectoryElementsStream(directoryUuid, userId, types, false);
+    }
```

Comment:
Maybe rename this method to make it clearer that it will return only not stashed elements